src/main/java/publishers/EmailPublisher.java

```diff
@@ -32,6 +32,12 @@
 
 public class EmailPublisher {
 
+	private Properties props2;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "props2" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1068)